api_reference/source/conf.py

```diff
@@ -1,53 +1,197 @@
+"""
+Configuration file for the Sphinx documentation builder.
+
+This file only contains a selection of the most common options. For a full
+list see the documentation:
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
+
+# -- Path setup --------------------------------------------------------------
 import os
 import sys
 
 sys.path.insert(0, os.path.abspath("../../"))
 sys.path.insert(0, os.path.abspath("../../src/phoenix"))
-# sys.path.insert(0, os.path.abspath('../packages/phoenix-evals/src/phoenix/evals'))
+sys.path.insert(0, os.path.abspath('../packages/phoenix-evals/src/phoenix'))
+
+# -- Generation setup --------------------------------------------------------
+
+# Define which modules, classes, and their respective methods to include
+INCLUDE_MEMBERS = {
+    'inferences.inferences': {
+        'Inferences': ['__init__']
+    },
+    # Add more here:
+    # 'module_name': {
+    #     'class_name': ['method_to_include'],
+    #     'class_name': ['method_to_include']
+    # }
+}
 
-
-# https://stackoverflow.com/questions/57610288/showing-only-private-methods-with-sphinx-autodoc
-# https://stackoverflow.com/questions/64351260/how-to-document-only-a-subset-of-functions-in-sphinx
 def include_only_tagged(app, what, name, obj, skip, options):
     inclusion_tag_format = ".. only:: {}"  # can be any pattern here, choose what works for you
     for tag in app.tags.tags:
         if obj.__doc__ is not None and inclusion_tag_format.format(tag) in obj.__doc__:
             return False
     return True
 
+def skip_member(app, what, name, obj, skip, options):
+    module_name = obj.__module__ if hasattr(obj, '__module__') else ''
+    class_name = obj.__class__.__name__ if hasattr(obj, '__class__') else ''
+    member_name = name.split('.')[-1]  # Get the last part of the name
+
+    # Check if the module is in the include list
+    if module_name in INCLUDE_MEMBERS:
+        if class_name in INCLUDE_MEMBERS[module_name]:
+            # Check if the member is in the include list for this class
+            return member_name not in INCLUDE_MEMBERS[module_name][class_name]
+        # Skip all other classes not specifically included
+        return True
+    # Skip all other modules not specifically included
+    return True
+
+def filter_rst(app, docname, source):
```

Comment:
is this from a specific source by chance? Might be good to link the source just so we can parse later on our own time

Reply:
No specific source code just used suggestions/research from different sources. Can link these sources in a later instructions markdown.